doc/explanation/api/examples/outliers_declarative.md

```diff
@@ -82,3 +82,40 @@ pn.Column(obj.param, obj.view)
 ```
 
 To support various domains, you can create hierarchies of classes encapsulating parameters and functionality across different object families. Parameters and code can inherit across classes as needed, without depending on any specific GUI library. This approach facilitates the maintenance of large codebases, all displayable and editable with Panel, adaptable over time. For a more complex illustration, refer to the [Attractors Panel app](https://examples.holoviz.org/gallery/attractors/attractors_panel.html) ([source](https://github.com/holoviz-topics/examples/tree/main/attractors)), and explore the Panel codebase itself for extensive usage of Param throughout the codebase.
+
+## Serving the Notebook
```

Comment:
I included this section because without it we could not really compare the result of the two approaches.

And the user could not get (VS Code) notebooks working.

Reply:
I really don't think this belongs here. These are explanation sections, not tutorials.

Reply:
In other words, we shouldn't recap templates and serving apps in every single example, especially if that isn't the focus.

Reply:
I'll leave it in this case.